src/java.base/share/classes/java/net/doc-files/net-properties.html

```diff
@@ -240,5 +242,57 @@ <H2>Address Cache</H2>
 <P>Since these 2 properties are part of the security policy, they are
 not set by either the -D option or the {@code System.setProperty()} API,
 instead they are set as security properties.</P>
+<a id="Unixdomain"></a>
+<H2>Unix domain sockets</H2>
+<p>
+Calling {@link java.nio.channels.ServerSocketChannel#bind(SocketAddress,int) ServerSocketChannel.bind}
+with a {@code null} address parameter will bind to an <i>automatically assigned</i> socket address.
+For Unix domain sockets, this means a unique path in some predefined system temporary directory.
+<P>There are a number of system (and networking) properties that affect this behavior.
```

Comment:
"There are are a number of system properties ..." follows directly from the previous sentence so I assume you don't want a paragraph break here.

Reply:
Ok, that's fine.